package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "playkit-js",
   "version": "0.1.0",
-  "main": "dist/playkit.js",
```

Comment:
If you remove the main then any library that uses Playkit will have to use the specific path when importing